### DISCUSSION TITLE
post/dictionary-bot/

### DISCUSSION THREAD

**Discussion by giscus** [kakaopay/techlog-comments#23] [Announcements]
# post/dictionary-bot/

회사 업무에 필요하지만 없었던 기능을 만들어서 개선한 경험을 공유합니다.

https://tech.kakaopay.com/post/dictionary-bot/

**Comment by unme88** [comment]
덕분에 잘쓰고 있습니다 👍👍

**Comment by pay-dan-dy** [comment]
용어 검색 시 편하게 잘 쓰고 있어요^^
